applications/POMP.md

```diff
@@ -0,0 +1,90 @@
+# POMP (Prove of My Possession)
+
+- **Team Name:** POMP
+- **Payment Address:** 0xfBa1C39f30d6A1D1C35F9551c0f22a0ec6FD30A5 (Ethereum USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+### Overview
+
+Cryptocurrencies are digital coins and tokens that represent real-world value without relying on a central issuer. Instead of going through a bank or a government, cryptocurrencies rely on cryptography, public ledgers, and blockchain technology to verify ownership. It’s playing a more and more important role in the real world. Let's just look at the numbers. Bitcoin had a market capitalization of $510 Billion on June 8, 2023. On the same day, Ethereum had a market capitalization of $220 Billion.
+
+However, it’s not easy to prove that you are the owner of one specific blockchain account. Because you have to import a private key to a wallet and sign a signature to prove that the private key is matching to the account’s public key. Additionally, the more a private key is imported, the more it opens up the threat vector for either intentional or unintentional damage.
+
+Many individuals and companies prefer to have an additional layer of asset proof to keep their privacy. That’s where the POMP(Prove of My Possession) comes to play.
+
+### Project Details
+
+The POMP is a decentralized service that combines zero-knowledge technology, users can easily generate encrypted proof keys while maintaining the privacy of their crypto assets. After generated, the user can prove the ownership of the crypto asset to any third party without inputting any private key.
+
+Here is the basic workflow for generating:
```

Comment:
So if I understand correctly, the user has to have Manta wallet in order to use the tool, correct?

Reply:
Yes, the user has to have a Manta wallet to generate the zero-knowledge data.